feat(loginPage)User should  change their passwords every x amount of time [187364810]

shebz2023 opened the pull request:
**what does this PR do?**

this Pr adds an interface that tells users with expired password to change their password 
it achieves this by navigating users with expired passwords to the change page and there they can find a link to change their password ,all this is for their security.

**what are relevant pivotal tracker id?**
187364810

1➡️2

![Screenshot 2024-06-28 202440_Combine](https://github.com/atlp-rwanda/e-commerce-serge-30-fn/assets/154588570/5016a52c-b579-450f-bb63-26f111010b61)

martinemahirwe reviewed (dismissed):
Good work @shebz2023

Sagastein reviewed (dismissed):
LGTM 🆗

SergeInGithub commented on src/pages/authpages/RequestPwdChangePage.tsx:
```diff
@@ -0,0 +1,15 @@
+export const RequestPwdChange = () => {
+  return (
+    <div className="flex bg-white">
+      <div 
+        className="hidden lg:block lg:w-1/2 bg-cover bg-center h-screen" role='image'
+        style={{ backgroundImage: `url(${leftImage})` }}
+      ></div>
```
```suggestion
      <div 
        className="hidden lg:block lg:w-1/2 bg-cover bg-center h-screen" role='image'
        style={{ backgroundImage: `url(${leftImage})` }}
      />
```

SergeInGithub reviewed (changes requested):
Address the minor change below and you're good

shebz2023 committed:
ft(login.page)users should change their password every-x-amount of time
after 60 days the password is set to expire this commit
create a renew form
a renew page

187364810

SergeInGithub reviewed (approved):
LGTM
